runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayUtil.java

```diff
@@ -341,6 +341,10 @@ public static void setGraphicProperties(Graphics2D graphics)
 			case TOP_RIGHT:
 				result.x = result.x - dimension.width;
 				break;
+			case ABOVE_CHATBOX_RIGHT:
```

Comment:
This case can be merged with BOTTOM_RIGHT case that is doing fallthrough to TOP_RIGHT case, end result is exactly same as inside of this new case.